.lintstagedrc

```diff
@@ -0,0 +1,14 @@
+{
+    "*.json": [
+      "prettier --write",
+      "git add"
+    ],
+    "*.(tsx|ts)": [
```

Comment:
Minor, but this can be one glob, for JS and TS, `*.{js,jsx,ts,tsx}`. See https://github.com/nickytonline/www.iamdeveloper.com/blob/master/package.json#L38

@zkat, tt might be out of the scope of this PR, but prettier handles markdown as well. I'd say let prettier handle as much of the formatting as possible. Thoughts?

```
"*.md": [
	"prettier --write",
	"git add"
]
```

Reply:
Same goes for rust, but I don't have any experience with rust. Do they have other packages for formatting rust code?

Reply:
Rust uses https://github.com/rust-lang/rustfmt

Reply:
> I'd say let prettier handle as much of the formatting as possible.

I'm down